Packages/manifest.json

```diff
@@ -18,6 +18,7 @@
     "com.unity.textmeshpro": "3.0.6",
     "com.unity.timeline": "1.6.4",
     "com.unity.ugui": "1.0.0",
+    "com.unity.vectorgraphics": "https://github.com/icosa-gallery/com.unity.vectorgraphics.git#open-brush",
```

Comment:
Should this move to the new icosa-mirror org?

Reply:
oh, i didn't see we'd also forked that! yep sure.

Reply:
SVGs are now written to png on export.